CHANGELOG.md

```diff
@@ -31,26 +36,32 @@
 - [VM/WarmMigration] Feature is for PREMIUM and SOURCE users (PR [#7514](https://github.com/vatesfr/xen-orchestra/pull/7514))
 - [VMWare/Migration] Handle raw CDROM
 - [XOSTOR Creation] The host toolstack is now restarted after installing the XOSTOR dependencies (PR [#7437](https://github.com/vatesfr/xen-orchestra/pull/7437))
+- [VMWare/Migration] Alignment of the end of delta on older ESXi (PR [#7487](https://github.com/vatesfr/xen-orchestra/pull/7487))
+- [Backup] Fix `no object with uuid or opaqueref` when running a health check (PR [#7467](https://github.com/vatesfr/xen-orchestra/pull/7467))
+- [Backup] Fix `task has already ended` when running a health check in a mirror backup (PR [#7467](https://github.com/vatesfr/xen-orchestra/pull/7467))
+- [Backup] Fix health check being stuck when using a different, non shared health check SR (PR [#7467](https://github.com/vatesfr/xen-orchestra/pull/7467))
 
 ### Released packages
 
 - @xen-orchestra/fs 4.1.6
+- vhd-lib 4.9.2
 - xen-api 3.0.0
 - @vates/nbd-client 3.0.1
 - @vates/otp 1.1.0
 - @xen-orchestra/xapi 5.0.0
-- @xen-orchestra/backups 0.46.0
+- @xen-orchestra/backups 0.46.1
 - @xen-orchestra/backups-cli 1.0.16
 - @xen-orchestra/cr-seed-cli 1.0.1
 - @xen-orchestra/immutable-backups 1.0.3
 - @xen-orchestra/self-signed 0.2.1
-- @xen-orchestra/proxy 0.27.0
-- @xen-orchestra/vmware-explorer 0.4.2
+- @xen-orchestra/proxy 0.27.1
+- @xen-orchestra/vmware-explorer 0.6.0
 - xapi-explore-sr 0.4.3
+- xo-acl-resolver 0.5.0
 - xo-cli 0.27.0
-- xo-server 5.139.0
+- xo-server 5.140.0
 - xo-server-load-balancer 0.9.0
-- xo-web 5.141.0
+- xo-web 5.142.0
```

Comment:
https://git.vates.tech/vates/company/wiki/XO-Release-process-Technical-release#update-changelog

`If a previous technical release has been done on the same packages, remove the old entries and place the new ones at the bottom of the list.`